azuredeploy.json

```diff
@@ -410,7 +410,7 @@
                 "Microsoft.Resources/deployments/redisTemplate",
                 "Microsoft.Resources/deployments/blobStorageAccountTemplate"
             ],
-            "name": "controllerTemplate",
+            "name": "pid-738e3eec-68d4-4667-8377-c05c77c21f1b",
```

Comment:
Wouldn't this obfuscate the deployment name in the customer's resource group deployment? I think I saw this name appearing as the deployment name, and I'm afraid this change will prevent us from identifying the sub-template deployment in the deployment resource group's deployments list...

Reply:
It will obfuscate that one template, yes. It won't obfuscate others. When submitting this I did make a mental note that we should probably have a "dummy" resource that can be this identifier. Do you know if that is possible? That is, an ARM resource deployment that doesn't actually do anything other than register this PID.

Reply:
It will be used as the deployment name, but won't change anything other than the name - the deployment operation will still be available as always.

You can also use an empty resource:

      "type": "Microsoft.Resources/deployments",
      "properties": {
        "mode": "Incremental",
        "template": {
          "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
          "contentVersion": "1.0.0.0",
          "resources": []
        }
      }